### PR TITLE
add openlmis component

### PR DESCRIPTION
**Pull request by Andati**

### CODE REVIEW

File: modules/connector/applicationContext-connector.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
 		<context:exclude-filter type="regex"
 			expression="org.opensrp.connector.rapidpro..*" />
     </context:component-scan>
+	<context:component-scan base-package="org.smartregister.stock.openlmis" />
```
Comment (githengi) [inline modules/connector/applicationContext-connector.xml:29]: This should be optional, define this inside an openlmis profile

File: modules/core/persistence_postgres.xml [hunk 2]
```diff
@@ -24,6 +24,7 @@
 		<context:exclude-filter type="regex"
 			expression="org.opensrp.connector..*" />
 	</context:component-scan>
+	<context:component-scan base-package="org.smartregister.stock.openlmis" />
```
Comment (githengi) [inline modules/core/persistence_postgres.xml:27]: This should be optional, define this inside an openlmis profile

File: modules/connector/applicationContext-connector.xml (hunk 1 above)
Comment (Andati) [reply modules/connector/applicationContext-connector.xml]: done

File: modules/core/persistence_postgres.xml (hunk 2 above)
Comment (Andati) [reply modules/core/persistence_postgres.xml]: removed this

**Comment by ndegwamartin** [comment]
Looks good @Andati , please bump up the artifact version

**Comment by Andati** [comment]
Looking at a few recent merges looks like config has no artifact version

**Review by qiarie — APPROVED** [review]

**Pull request merged**